e2e/tests/upgrades/upgrade_test.go

```diff
@@ -24,8 +24,11 @@ import (
 	"github.com/cosmos/ibc-go/e2e/testvalues"
 	controllertypes "github.com/cosmos/ibc-go/v6/modules/apps/27-interchain-accounts/controller/types"
 	icatypes "github.com/cosmos/ibc-go/v6/modules/apps/27-interchain-accounts/types"
+	v7 "github.com/cosmos/ibc-go/v6/modules/core/02-client/migrations/v7"
```

Comment:
maybe the alias should be `v7migrations` instead of just `v7` to make it explicit in the tests. The same way we have `v7upgrades`